tests/Query/Filter/PathField.test.ts

```diff
@@ -37,4 +108,34 @@ describe('path', () => {
         testTaskFilterForTaskWithPath(filter, '/some/path/file.md', false);
         testTaskFilterForTaskWithPath(filter, '/other/path/file.md', true);
     });
+
+    it('by path (regex matches)', () => {
+        // Arrange
+        const filter = new PathField().createFilterOrErrorMessage(
+            'path regex matches /w.bble/',
+        );
+
+        // Assert
+        expect(filter).toBeValid();
+        expect(filter).toMatchTaskWithPath('/some/path/wibble.md');
+        expect(filter).toMatchTaskWithPath('/some/path/wobble.md');
+        expect(filter).not.toMatchTaskWithPath('');
+        expect(filter).not.toMatchTaskWithPath('/some/path/WobblE.md'); // confirm case-sensitive
+        expect(filter).not.toMatchTaskWithPath('/other/path/file.md');
+    });
+
+    it('by path (regex matches) with flags', () => {
+        // Arrange
+        const filter = new PathField().createFilterOrErrorMessage(
+            'path regex matches /w.bble/i',
+        );
+
+        // Assert
+        expect(filter).toBeValid();
+        expect(filter).toMatchTaskWithPath('/some/path/wibble.md');
+        expect(filter).toMatchTaskWithPath('/some/path/wobble.md');
+        expect(filter).not.toMatchTaskWithPath('');
+        expect(filter).toMatchTaskWithPath('/some/path/WobblE.md'); // confirm case-insensitive (flag)
+        expect(filter).not.toMatchTaskWithPath('/other/path/file.md');
```

Comment:
Is there any value in repeating all the same cases as from the previous test? 
Could this be pruned back to just tests that test the case-insensitive-ness?

Reply:
It doesn't hurt to show that the flag changed no other behavior. That RegExp is used under the hood is an implementation detail.